miden-vm/src/cli/debug/mod.rs

```diff
@@ -57,7 +57,7 @@ impl DebugCmd {
             "masp" => {
                 (get_masp_program(&self.program_file)?, Arc::new(DefaultSourceManager::default()))
             },
-            "masm" => get_masm_program(&self.program_file, &libraries, true)?,
+            "masm" => get_masm_program(&self.program_file, &libraries, true, None)?,
```

Comment:
let's add a `--kernel` flag to `miden debug`, `miden prove`, and `miden verify` as well.

For `miden verify` [here](https://github.com/0xMiden/miden-vm/blob/f3a52ca8535df570f081b5ed2c8f61de8d184753/miden-vm/src/cli/verify.rs#L53-L54) we'll need to load the kernel, and just extract `KernelLibrary.kernel`.